reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -123,10 +134,27 @@ Aliases:
 Required: True
 Position: 1
 Default value: None
-Accept pipeline input: True (ByValue)
+Accept pipeline input: False
```

Comment:
You updated the wrong one.  This one should be reverted.  Line 154 below should be `false`

Reply:
Oops. Sorry, well spotted. Should be fixed now.